shopify/customized-bundle-template/web/components/UpdateForm.tsx

```diff
@@ -26,16 +26,16 @@ const SkeletonForm = () => {
   return (
     <Card>
       <SkeletonPage primaryAction>
-        <Form>
+        <Form onSubmit={() => {}}>
```

Comment:
Do we need the form component in a loading skeleton?

Reply:
True